### DISCUSSION TITLE
@layer base  -  no longer working when mode: 'jit'

### DISCUSSION THREAD

**Discussion by schlichtanders** [tailwindlabs/tailwindcss#5835] [Help] [answered]
I just tried enabling 'jit' mode, because I wanted to use `before:` and `after:` variants.

I am using tailwind cli to compile my css. Reading standard troubleshootings I think this should work in principle.

Observed Behaviour:
* when I enable jit mode, all css defined in `@layer base { ... }` do not apply, making jit mode impossible to use for me.
* `@layer components` works fine though
* I do not get syntax errors when using `before:` and `after:`, so it seems jit mode did something.

Expected Behaviour:
* enabling jit mode should also work with `@layer base { ... }`

Any help is highly appreciated, as I would like use `before:` and `after:` for my styling

-----------------------

EDIT:
I am using this tailwind cli command:
```
npx tailwindcss -i _css/my.css -o _css/tailwind.css --watch
```
and this tailwind config:
```
const defaultTheme = require('tailwindcss/defaultTheme')
module.exports = {
  mode: 'jit',
  purge: [
    // ... 
  ],
  darkMode: false,
  theme: {
    extend: {
      fontFamily: {
        sans: ['MySpecialFont', ...defaultTheme.fontFamily.sans]
      }
    },
  },
  plugins: [],
}
```

**Comment by alukard111** [comment, marked as answer]
The same problem came up.
Removing @layer base {} helped
Here they advised -
https://github.com/tailwindlabs/tailwindcss-jit/issues/66#issuecomment-800436823

**Reply by schlichtanders** [reply]
this really worked for me!! thank you very much
